### ISSUE TITLE
Issue on page /docs/e_raster_rasterize.html

### ISSUE THREAD

**Issue by CaspianMaster** [mmann1123/pyGIS#57]
As i can see, if all_touched set is true then include any cell that geometry boundary intersects. But in Fig. 58  bottom square under circle is not included. Can you tell what am i misunderstanding?

**Comment by mmann1123** [comment]
You are correct, I posted the wrong image and haven't gotten around to fixing it.

**Comment by mmann1123** [comment]
Thanks for catching this. Sorry for the late response
